### DISCUSSION TITLE
给办理退款：已经两个月了

### DISCUSSION THREAD

**Discussion by Niejingchun** [getlantern/lantern-forum-cn#1092] [General]
给办理退款。已经两个月了。原贴如下：https://github.com/getlantern/lantern-forum-cn/discussions/967

**Comment by MaxMKMax** [comment]
@Niejingchun 您好. 上一个帖子我们的支付系统团队有回复您了. 您的支付单据无法让我们进行退款. 他说系统里根本找不到. 我也帮您问过 3 次了. 请问您有别的单据吗?

**Reply by Niejingchun** [reply]
我查了下，google play里有一个这个。这个可以吗？我的支付卡号是Discovery的卡：6011007504550624。
![Uploading 微信图片_20241221191946.jpg…]()

**Reply by Niejingchun** [reply]
![微信图片_20241221191946](https://github.com/user-attachments/assets/8d40af4c-36c2-49a2-b6c5-20d557c7eb3e)

**Reply by Niejingchun** [reply]
我提交这个支付凭证，可以给我办理退款了吧

**Comment by MaxMKMax** [comment]
@Niejingchun 您好. 我帮您问问看. 再次帮您提交问题上去.

**Comment by MaxMKMax** [comment]
@Niejingchun 已经帮您提交好了. 您留意一下客服的邮件哦. 谢谢您的耐心等待.
